### DISCUSSION TITLE
Is mangohud possible?

### DISCUSSION THREAD

**Discussion by lietuv1s** [rfvgyhn/min-ed-launcher#115] [Q&A] [answered]
ED in Steam (Ubuntu) works with:
`gnome-terminal -- ./MinEdLauncher %command% /autorun /autoquit /edo -nointro`
And I tried placing mangohud everywhere with no luck.

**Comment by rfvgyhn** [comment]
Yes. I have it globally enabled via env var, but manually specifying mangohud in the launch options also works for me.

`mangohud gnome-terminal -- ./MinEdLauncher %command% /edh4 /autoquit /autorun`
`gnome-terminal -- mangohud ./MinEdLauncher %command% /edh4 /autoquit /autorun`

**Comment by lietuv1s** [comment]
Thank you for your answer. Mangohud does appear (both options) when starting at first where it goes to black screen (shows gorgeous FPS above 1k). But goes away right when intro starts and never shows up again. Tried windowed mode too. I dont have it globally enabled. Is that my only option?

**Comment by demonbane** [comment, marked as answer]
If you really want to avoid enabling it globally, you can try:
```
MANGOHUD=1 gnome-terminal -- mangohud ./MinEdLauncher %command% /edh4 /autoquit /autorun
```
Assuming that none of the intermediate processes clear the environment that **should** have the same effect as enabling it globally. (you could probably even leave out the `mangohud` part altogether as `libvulkan.so` should still pick it up either way)

**Reply by lietuv1s** [reply]
"MANGOHUD=1" part worked like a charm. Mangohud showed up and stayed! Thank you all!
`MANGOHUD=1 gnome-terminal -- ./MinEdLauncher %command% /autorun /autoquit /edo`
